run.py

```diff
@@ -131,15 +113,12 @@ def get_watson_online_store():
 
         # Instantiate Watson Discovery client.
         # - only give a url if we have one (don't override the default)
-        discovery_kwargs = {
-            'version': '2019-04-30',
-            'iam_apikey': discovery_iam_apikey
-
-        }
-        if discovery_url:
-            discovery_kwargs['url'] = discovery_url
-
-        discovery_client = DiscoveryV1(**discovery_kwargs)
+        discovery_client = DiscoveryV1(
+            version='2019-11-22',
+        )
+        discovery_client.set_service_url(discovery_url)
```

Comment:
I don't think this should be necessary, but can't confirm right now.  Pretty sure other apps do this by setting the well-known env var that the SDK recognizes to do this.